Resources/Prototypes/_Goobstation/Roles/Antags/devil.yml

```
@@ -19,7 +20,11 @@
     time: 18000 # 5 hours
   - !type:RoleTimeRequirement
     role: JobLawyer
-    time: 3600 # 1 Hour
+    time: 18000 # 5 Hours
+    role: JobNanotrasenRepresentative
```

YAMLlint (1.35.1)

[error] duplication of key "role" in mapping

(key-duplicates)